### PR TITLE
Removing "so" from Docs

### PR DESCRIPTION
**Pull request by DarthOstrich**
*Issue #328 , if available:*

*Description of changes:*
Removing "so" from the docs

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by DarthOstrich** [comment]
I'm not sure what other people have done, so I'll do a second pass on this after the PRs close.

**Comment by dbanksdesign** [comment]
We just merged a large PR with documentation edits. @DarthOstrich could you rebase and fix any conflicts please. Thank you!

**Comment by DarthOstrich** [comment]
Will do! I'll take care of it tomorrow.

File: docs/architecture.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Style Dictionary takes all the files it found and performs a deep merge. This al
 
 ## 4. Iterate over the platforms
 
-For each platform defined in your [config](config.md), Style Dictionary will do a few steps to get it ready to be consumed on that platform. Everything that happens in a platform is non-destructive; you don't need to worry about one platform affecting another.
+For each platform defined in your [config](config.md), Style Dictionary will do a few steps to get it ready to be consumed on that platform. You don't need to worry about one platform affecting another because everything that happens in a platform is non-destructive
```
Comment (dbanksdesign) [inline docs/architecture.md]: Much better!

File: docs/config.md [hunk 2]
```diff
@@ -38,9 +38,9 @@ You can find out more about creating configurations in JS in our documentation a
 | Attribute | Type | Description |
 | :--- | :--- | :--- |
 | include | Array[String] (optional) | An array of path [globs](https://github.com/isaacs/node-glob) to Style Dictionary property files that contain default styles. The Style Dictionary uses this as a base collection of properties. The properties found using the "source" attribute will overwrite properties found using include. |
-| source | Array[String] | An array of paths that can be [globs](https://github.com/isaacs/node-glob) to JSON files that contain style properties. This allows you to separate your properties into multiple files as Style Dictionary will do a deep merge of all of the JSON files to create the dictionary. |
+| source | Array[String] | An array of path [globs](https://github.com/isaacs/node-glob) to JSON files that contain style properties. The Style Dictionary will do a deep merge of all of the JSON files, allowing you to separate your properties into multiple files. |
```
Comment (dbanksdesign) [inline docs/config.md]: Love it.

File: docs/config.md [hunk 3]
```diff
@@ -38,9 +38,9 @@ You can find out more about creating configurations in JS in our documentation a
...
 | platforms | Object | An object containing platform config objects that describe how the Style Dictionary should build for that platform. You can add any arbitrary attributes on this object that will get passed to formats and actions (more on these in a bit). This is useful for things like build paths, name prefixes, variable names, etc.  |
-| platform.transforms | Array[String] (optional) | An array of [transforms](transforms.md) to be performed on the style properties object. These transform the properties in a non-destructive way as each platform can transform the properties in a unique manner. Transforms to apply sequentially to all properties. Can be a built-in one or you can create your own. |
+| platform.transforms | Array[String] (optional) | An array of [transforms](transforms.md) to be performed on the style properties object. These will transform the properties in a non-destructive way, allowing each platform can transform the properties. Transforms to apply sequentially to all properties. Can be a built-in one or you can create your own. |
```
Comment (chazzmoney) [inline docs/config.md]: > allowing each platform can transform the properties. 

allowing each platform to transform the properties.

File: docs/formats.md [hunk 4]
```diff
@@ -33,7 +33,7 @@ There is an extensive (but not exhaustive) list of [included formats](#pre-defin
 
 ### Format configuration
 
-Formats can take configuration to make them more flexible. This allows you to re-use the same format multiple times with different configurations or to allow the format to use data not defined in the tokens themselves. To configure a format, add extra attributes on the file object in your configuration like the following:
+Formats can take configuration to make them more flexible. This allows you to re-use the same format multiple times with different configuration or to allow the format to use data not defined in the tokens themselves. To configure a format add extra attributes on the file object in your configuration as written below:
```
Comment (chazzmoney) [inline docs/formats.md]: I like the original better, actually, thought the first sentence is confusing. I do like your change at the end to "as written below". So maybe this:

Formats are flexible and may accept configuration options. This allows you to re-use the same format multiple times with different configurations or to allow the format to use data not defined in the tokens themselves. To configure a format, add extra attributes on the file object in your configuration as written below:

@dbanksdesign thoughts?

Comment (dbanksdesign) [reply docs/formats.md]: love it :shipit:

**Comment by dbanksdesign** [comment]
@DarthOstrich any update on the 2 changes requested? We would love to get this in!

**Comment by DarthOstrich** [comment]
Pretty sure I address those in the two commits? If I'm missing something I can adjust it.

**Comment by dbanksdesign** [comment]
Oh sorry, when I clicked the links to the changed files those comments were on Github showed me a previous version so I thought they weren't done yet. LGTM!

**Pull request merged**